msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -1,5 +1,14 @@
-|Microsoft 365 app host| frame-ancestor permission|
+> [!WARNING]
+> Microsoft's cloud services, including web versions of Teams (*teams.microsoft.com*), Outlook (*outlook.com*), and Microsoft 365 (*microsoft365.com*) domains are migrating to the new *cloud.microsoft* domain by June 2024. We recommend you to perform the following steps before June 2024 to ensure your app renders on the Teams web client:
```

Comment:
How about tightening up the language here too, similar to the "what's new" blurb:

Microsoft's cloud services, including web versions of Teams (*teams.microsoft.com*), Outlook (*outlook.com*), and Microsoft 365 (*microsoft365.com*) are migrating to the new *cloud.microsoft* domain. Update your app with the following steps before June 2024 to prevent your app from losing critical functionality:

Reply:
Done